src/ext/processObliv/processObliv.ml

```diff
@@ -49,6 +49,9 @@ let oblivIntTarget = ref dummyType
 let oblivShortTarget = ref dummyType
 let oblivLongTarget = ref dummyType
 let oblivLLongTarget = ref dummyType
+let oblivFloatTarget = ref dummyType (*CHANGE*)
```

Comment:
As elsewhere, these (*CHANGE*) markers should probably be annotated with the reason.